### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by AsalaKM**
**Back-End**
-view the reviews .
-view comments that related to reviews.

**Front-End**
-Adding contact-us page and view the contact form with front end validations.
-Adding about us page.
-Editing some styles.

### CODE REVIEW

**Review by asmaa-ezz — APPROVED** [review]

File: public/js/home.js [hunk 1]
```diff
@@ -1,5 +1,25 @@
+/* global document */
 const search = document.getElementById('search');
 const list = document.getElementById('data');
+// const reviewsSection = document.getElementsByClassName('reviews-section');/
+const buttonComment = document.getElementsByClassName('button-comment')[0];
+
+// buttonComment.addEventListener('click', () => {
```
Comment (FarahZaqout) [inline public/js/home.js]: Remove commented code.

File: src/model/database/db_build.sql [hunk 2]
```diff
@@ -44,4 +44,20 @@ CREATE TABLE comment (
 	content TEXT NOT NULL
 );
 
+INSERT INTO users (email, password, type) VALUES ('asmaa@gmail.com', '000', 'person');
```
Comment (FarahZaqout) [inline src/model/database/db_build.sql]: No sql commands allowed in build.sql file.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]
Pull request is titled reviews, yet it has a lot of files unrelated to reviews.

File: package.json [hunk 3]
```diff
@@ -7,8 +7,14 @@
     "test": "test"
   },
   "scripts": {
+<<<<<<< HEAD
```
Comment (FarahZaqout) [inline package.json]: This is an unresolved conflict.

Use travis.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by donia98 — APPROVED** [review]

File: public/js/contact_us.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/* global document */
+
+const btnForm = document.getElementsByClassName('btn')[0];
+
+btnForm.addEventListener('click', () => {
+  // alert('Thanks For Your Sending');
```
Comment (FarahZaqout) [inline public/js/contact_us.js]: This function does nothing.

File: src/views/partials/head.hbs [hunk 5]
```diff
@@ -3,10 +3,17 @@
 <meta http-equiv="X-UA-Compatible" content="ie=edge">
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
 <link href="https://fonts.googleapis.com/css?family=Pacifico|Inconsolata|Roboto+Mono" rel="stylesheet">
+<<<<<<< HEAD
```
Comment (FarahZaqout) [inline src/views/partials/head.hbs]: Unresolved conflict.

File: src/views/contact_us.hbs [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+			<div class="contact-section" >
+				<h2><span>voyage</span>location</h2>
+				<div class="map-section">
+					<iframe id="map" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d435859.4344905242!2d34.668202450021134!3d31.409942603049434!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x14fd844104b258a9%3A0xfddcb14b194be8e7!2z2YLYt9in2Lkg2LrYstmR2Kk!5e0!3m2!1sar!2s!4v1536931969076" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+					<div class="map-rgba"></div>
+				</div>
+				<div id="get-in-touch" class="contact">
+					<h2 class="contact-title">get in touch</h2>
+					<div class="container">
+							<div class="contact-form">
```
Comment (FarahZaqout) [inline src/views/contact_us.hbs]: this is too much indentation. Make sure no eslint warnings are in your files.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+- node
```
Comment (FarahZaqout) [inline .travis.yml]: add new line at the end of your files.

File: public/css/about_us.css [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    display: grid;
+    grid-template-columns: 60% 40%;
+}
+.about-idea .imgs{
+
+}
+.about-idea .imgs img{
+    width: 100px;
+    height: 100px;
+}
```
Comment (FarahZaqout) [inline public/css/about_us.css]: add new line.

File: public/css/contact_us.css [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+    font-weight: bold;
+    font-size:18px;
+    transition:all 0.5s ease 0s;
+    
+}
+.contact-icon-box:hover p{
+    font-size:15px;
+}
+      
+}
```
Comment (FarahZaqout) [inline public/css/contact_us.css]: add new line

**Review by FarahZaqout — CHANGES_REQUESTED** [review]
Make sure no eslint warnings are up.

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**